src/dms-preview/azext_dms/_help.py

```diff
@@ -48,23 +50,23 @@
         - name: --source-platform
           type: string
           short-summary: >
-            The type of server for the source database. The supported types are: SQL, MySQL, PostgreSQL.
+            The type of server for the source database. The supported types are: SQL, MySQL, PostgreSQL, MongoDB.
         - name: --target-platform
           type: string
           short-summary: >
-            The type of server for the target database. The supported types are: SQLDB, AzureDbForMySql, AzureDbForPostgreSQL.
+            The type of server for the target database. The supported types are: SQLDB, AzureDbForMySql, AzureDbForPostgreSQL, MongoDB.
```

Comment:
Should "MongoDB" be a "CosmosDB"?

Reply:
Nope. See my other reply to your comment. The type for both source and target are MongoDB.